juniper_official/chassis/check-fan-state-rpm.rule

```diff
@@ -0,0 +1,146 @@
+/*
+ * Moniters fan state and RPM percentage and notifies when anomalies are found.
+ * 
+ */
+healthbot {
+    topic chassis {
+        rule check-fan-state-rpm {
+            /*
+             * Moniters fan state and RPM percentage and notifies via dashboard, red color 
+             * when threshold is exceeded else green.
+			 *
+             * Use fan-name as key for rule.
+             */		
+            keys fan-name;
+            synopsis "Fan KPI";
+            description "Checks Fan state and RPM percentage";
```

Comment:
Replace description to "Collects chassis environment statistics periodically and notifies anomalies when fan status is NOK"